web-client/src/presenter/computeds/messageDocumentHelper.test.js

```diff
@@ -290,6 +290,35 @@ describe('messageDocumentHelper', () => {
     expect(result.showRemoveSignatureButton).toEqual(false);
   });
 
+  it('return showApplySignatureButton false and showRemoveSignatureButton false for an internal user and an SDEC document', () => {
+    applicationContext.getCurrentUser.mockReturnValue(docketClerkUser);
+
+    const result = runCompute(messageDocumentHelper, {
+      state: {
+        ...getBaseState(docketClerkUser),
+        caseDetail: {
+          correspondence: [],
+          docketRecord: [],
+          documents: [
+            {
+              documentId: '123',
+              entityName: 'Document',
+              eventCode: 'SDEC',
+              isDraft: true,
+              signedAt: '2020-06-25T20:49:28.192Z',
```

Comment:
would a draft document have a signedAt?

Reply:
It can, but draft documents can't be served.